ena-submission/scripts/deposition_dry_run.py

```diff
@@ -126,7 +126,7 @@ def local_ena_submission_generator(
             "curl -X POST $ena_submission_url "
             "-u '$ena_submission_username:$ena_submission_password' "
             "-F 'SUBMISSION=@project/submission.xml' -F 'PROJECT=@project/project.xml'"
-            " --max-time 10 > {output}"
+            " --max-time 50 > {output}"
```

Comment:
this isn't related but I noticed the timeout is too short now when testing (it does seem like the increase in latency of the ENA endpoint was a recent change as I used to submit with these commands quite a bit)